app.rb

```diff
@@ -1,37 +1,58 @@
 require 'sinatra'
 require './lib/profile'
+require './lib/we_got_coders/content.rb'
 
-set :application_id, 'EDIT_ME'
-set :secret, 'EDIT_ME'
+set :application_id, '80054c1b3206110a5743c2f41c7a035b3f882889e4ff92551a11e522565a911a'
+set :secret, '0e2fd70d0af46a0755aa545a1435edc5e699df8075e089eef200c8d5bc7d2387'
 set :redirect_uri, 'http://localhost:4567/callback'
 set :site_url, 'http://wegotcoders.com'
 set :session_secret, 'secret'
 enable :sessions
 
+include Sinatra::OauthRoutes
+
 get '/primes' do
-  # TODO - Can we make this dynamic?
-  limit = 100
 
-  # TODO - add your prime number solution in the primes.rb file.
+ 
+  params[:prime_sums] ? limit = params[:prime_sums].to_i : limit = 1
```

Comment:
You can also just use boolean operators here:
`limit = params[:prime_sums] || 1`
which allows you to have a default value if the parameter is nil